src/serlio/util/Utilities.cpp

```diff
@@ -28,6 +28,7 @@
 struct IUnknown;
 #	include <process.h>
 #	include <windows.h>
+#	include <shellapi.h>
```

Comment:
what is this include needed for? (also not alphabetically ordered, or does it have to be after `windows.h`?)

Reply:
i honestly don't know, i moved some incudes around due to the main change in this PR and suddenly the windows build broke... it must have been some dark include chain....

Reply:
shellapi.h needs to be included after windows.h